README.md

```diff
@@ -1,2 +1,47 @@
+
+## Source Aims and Conclusion
+With the sources I picked, none of them seem to have an overall stance or if SDV's are bad or good, Sources 1 and 5 primary points seem to draw out the problems that SDV's currently have, and then propose ways in which these problems could be fixed or improved, which is helpful to developers of these vehicles, paper 3 is very much the same but just outlines where SDV's could come under scrutiny in law terms and again points out how they could be improved to get around these issues. Paper 4 mearly outlines early SDV technology but is still helpful to see how far they've come since then, while Paper 6 goes much further in depth about the workings of SDV's and was useful for drawing conclusions about the current technology that is used in SDV's to further my conclusions. And paper 7 was useful as it outlines human ethical decisions in scenarios that SDV's would be put in.
+
+
+To conclude, all of the sources I have covered here do lend something to the conclusions that I have drawn over the course of my journal, I do think that SDV's have the capacity to make human-like "ethical" decisions but manufacturers will have to incorporate a number of the systems proposed in some of these papers and other that are being developed to make a SDV fully capable of making it's own decisions, along with the fact that the entire road system will need to be reworked for them, to make them function flawlessly it will be a long and expensive process, but I think one that is worth waiting for when looking at the number of lives that could be saved.
```

Comment:
Would be useful to have a source to back up the assertion about "the number of lives that could be saved" -- are there any studies on whether autonomous cars are safer than those driven by humans, and by how much?